GTG/gtk/editor/editor.py

```diff
@@ -836,13 +836,16 @@ def destruction(self, _=None):
         # process textview to make sure every tag is parsed
         self.textview.process()
 
-        # Save should be also called when buffer is modified
+        # Save because self.textview.process() only calls light_save(), which
+        # isn't guaranteed to actually save (see #1138)
+        self.save()
+
         # self.pengine.onTaskClose(self.plugin_api)
         # self.pengine.remove_api(self.plugin_api)
 
         tid = self.task.id
 
-        if self.is_new():
+        if self.task.is_new():
```

Comment:
```
    def is_new(self) -> bool:
        return self.task.is_new()
```
I wouldn't mind getting rid of that method eventually.

Reply:
Definitely, but watch out for other uses. Check out application.py at line 436

Reply:
Oh yeah I don't mean to just delete it now, just that it is now a wrapper around something that makes more sense, so we could replace the usages and eventually remove the method.